data-prepper-expression/build.gradle

```diff
@@ -38,7 +38,7 @@ jacocoTestCoverageVerification {
     violationRules {
         rule { //in addition to core projects rule - this one checks for 100% code coverage for this project
             limit {
-                minimum = 0.8 //keep this at 100%
+                minimum = 1.0 //keep this at 100%
```

Comment:
Wow, that was temporary! I was concerned we would forget to come back and fix it.